tests/cypress/e2e/create-post.test.js

```diff
@@ -83,7 +83,20 @@ describe('Command: createPost', () => {
       title: postTitle,
       content: postContent,
       beforeSave: () => {
-        cy.openDocumentSettingsPanel('Status & visibility');
+        // WP 6.1 renamed the panel name "Status & visibility" to "Summary".
+        cy.openDocumentSettingsSidebar('Post');
+        cy.get('body').then($body => {
+          let name = 'Status & visibility';
+          if (
+            $body.find(
+              '.components-panel__body .components-panel__body-title button:contains("Summary")'
+            ).length > 0
+          ) {
+            name = 'Summary';
+          }
+          cy.openDocumentSettingsPanel(name);
+        });
```

Comment:
The selector for the component hasn't changed, ie it is `.components-panel .edit-post-post-status` in both 6.0 and 6.1. Is it possible to use that instead?

Otherwise, maybe see if it's possible to catch the change in `openDocumentSettingsPanel` to maintain backward compatibility for other repos using the function. @cadic may have some thoughts here.

Reply:
Yes, I think it is better to catch the change in `openDocumentSettingsPanel` to get this maintained among other repos as well. what do you think @cadic @Sidsector9?